test/test_playbooks/domain_vars.yml

```diff
@@ -0,0 +1,8 @@
+---
+domain_name: "example.org"
+domain_dns_proxy: "centos7-devel.example.com"
+domain_locations:
+  - 'Default Location'
+domain_organizations:
+  - 'Default Organization'
```

Comment:
Please use the provided "Test <Taxonomy>" here.

Reply:
I'd rather keep all config-data for the test in one file.